include/IrrCompileConfig.h

```diff
@@ -140,6 +140,14 @@ the engine will no longer read .png images. */
 #undef _IRR_COMPILE_WITH_LIBPNG_
 #endif
 
+//! Define _IRR_COMPILE_WITH_GLI_ to enable compiling the engine using gli.
+/** This enables the engine to read ktx images. If you comment this out,
+the engine will no longer read .ktx images. */
+#define _IRR_COMPILE_WITH_GLI_
```

Comment:
nope make a CMAKE option and in the configure file, just like `IRR_BUILD_MITSUBA`